### PR TITLE
fix java bump changelog in v1.18.0 docs

### LINKED ISSUE #5576: Upgrading v1.18.0 documentation minor issue
V1.18 documentation https://sdk.operatorframework.io/docs/upgrading-sdk-version/v1.18.0/

Using the YAML string ‘|’ operator means that newlines in this string will Then in the bundle target we want to replace the flags passed to generate bundle with a reference to the BUNDLE_GEN_FLAGS above.

should be:
Then in the bundle target we want to replace the flags passed to generate bundle with a reference to the BUNDLE_GEN_FLAGS above.

### PR DESCRIPTION
**Pull request by jberkhahn**
This wasn't actually a typo, seems @jmrodri forgot to fill in his changelog fragment?

fixes #5576

### CODE REVIEW

**Comment by jberkhahn** [comment]
could @jmrodri take a look at this? the original PR is #5542 for reference

File: website/content/en/docs/upgrading-sdk-version/v1.18.0.md [hunk 1]
```diff
@@ -3,13 +3,24 @@ title: v1.18.0
...
+    chore: bump go 1.17, k8s 1.23, and kubebuilder 3.3
+    chore(deps): update to Quarkus SDK extension 3.0.2 and Quarkus 2.6.3
+    chore(deps): update to use Quarkus JOSDK extension 3.0.1
+    Remove useless file
+    chore: bump k8s 1.22.2 and kubebuilder 3.2
+    exposed exnpoints for micrometer metrics
+    modified the Quarkus operator SDK version and tutorial too
+    chore: bump fabric8 5.8.0 & quarkus 2.4.0
+    modified the doc file and removed file after generation
+    release: fix release script to understand release branches
```
Comment (camilamacedo86) [inline website/content/en/docs/upgrading-sdk-version/v1.18.0.md:19]: Should we not use here - as a prefix of all lines?

**Comment by jmrodri** [comment]
So what is this trying to fix? I had a changelog for my v0.2.0 and my v0.3.0 bumps.

* v0.2.0 - https://github.com/operator-framework/operator-sdk/pull/5542/files
* v0.3.0 - https://github.com/operator-framework/operator-sdk/pull/5588/files

So what's working?

**Comment by jberkhahn** [comment]
@jmrodri This PR is to fix the page in the upgrading-sdk docs, which is supposed to get populated from the changelog fragments, right? But currently there's a section that looks like changelog boilerplate where the bit from your PR is supposed to be.

**Review by jmrodri — APPROVED** [review]
/lgtm

**Pull request merged**